### ISSUE TITLE
Battery Warning

### ISSUE THREAD

**Issue by BrackoNe** [ydrive/citysynth-android-issues#24]
**Is your feature request related to a problem? Please describe.**
There is a potential problem if the user starts route tracing/recording, but the battery runs out, we would not have a complete route.

**Describe the solution you'd like**
There should be a prevention mechanism for starting a new task if the battery level is lower than 10%, but it should display a warning to the user if the recording is on. This warning should warn the user that the battery is running low and it is a good idea to finish the recording as soon as possible. Once the recording is finished, the app should prevent starting a new recording.
